cirrus/google_cloud/iam.py

```diff
@@ -218,7 +218,15 @@ def __init__(self, name):
         Args:
             name (str): The name of the Google role
         """
-        self.name = name
+        # if using a traditional role, remove the prefix for the name
+        # NOTE: Custom roles have a different prefix, and we will transparently
+        #       have that as the name since the prefix is dynamic (e.g. it changes
+        #       based on the project/org the custom role was defined in)
+        if name.strip().startswith(GooglePolicyRole.ROLE_PREFIX):
+            self.name = name[len(GooglePolicyRole.ROLE_PREFIX) :]
```

Comment:
if `name` starts with whitespace this won't work, i think you can't avoid doing `self.name = self.name.strip()` first :p

Reply:
oh, shoot :man_facepalming: ugh, yeah

Reply:
well, I can just `name = name.strip()`